README.md

```diff
@@ -101,9 +101,9 @@ Verify the installation by running:
 Once Ruby and JavaScript are installed, install Ruby dependencies by running:
 
     gem install bundle
-    bundle install
+    bundle install --path ./ruby_modules
```

Comment:
I believe Windows can handle any slashes. @purplecabbage can you confirm? I'll test